tests/countries/test_cabo_verde.py

```diff
@@ -0,0 +1,713 @@
+        name = "Dia da Liberdade e da Democracia"
+        self.assertHolidayName(name, (f"{year}-01-13" for year in range(2000, 2050)))
+        self.assertNoHolidayName(name, range(1976, 2000))
+
+    def test_heroes_day(self):
+        self.assertHolidayName(
+            "Dia da Nacionalidade e dos Heróis Nacionais",
+            (f"{year}-01-20" for year in range(1976, 2050)),
+        )
+
+    def test_ash_wednesday(self):
+        name = "Quarta-feira de Cinzas"
+        self.assertHolidayName(
+            name,
+            "2019-03-06",
+            "2020-02-26",
+            "2021-02-17",
+            "2022-03-02",
+            "2023-02-22",
+            "2024-02-14",
+            "2025-03-05",
+        )
+        self.assertHolidayName(name, range(1976, 2050))
+
+    def test_good_friday(self):
+        name = "Sexta-feira Santa"
+        self.assertHolidayName(
+            name,
+            "2019-04-19",
+            "2020-04-10",
+            "2021-04-02",
+            "2022-04-15",
+            "2023-04-07",
+            "2024-03-29",
+            "2025-04-18",
+        )
+        self.assertHolidayName(name, range(1976, 2050))
+
+    def test_easter_sunday(self):
+        name = "Páscoa"
+        self.assertHolidayName(
+            name,
+            "2019-04-21",
+            "2020-04-12",
+            "2021-04-04",
+            "2022-04-17",
+            "2023-04-09",
+            "2024-03-31",
+            "2025-04-20",
+        )
+        self.assertHolidayName(name, range(1976, 2050))
+
+    def test_workers_day(self):
+        self.assertHolidayName(
+            "Dia do Trabalhador", (f"{year}-05-01" for year in range(1976, 2050))
+        )
+
+    def test_international_childrens_day(self):
+        name = "Dia Mundial da Criança"
+        self.assertHolidayName(name, (f"{year}-06-01" for year in range(2005, 2050)))
+        self.assertNoHolidayName(name, range(1976, 2005))
+
+    def test_independence_day(self):
+        self.assertHolidayName(
+            "Dia da Independência Nacional", (f"{year}-07-05" for year in range(1976, 2050))
+        )
+
+    def test_assumption_day(self):
+        self.assertHolidayName("Dia da Assunção", (f"{year}-08-15" for year in range(1976, 2050)))
+
+    def test_all_saints_day(self):
+        self.assertHolidayName(
+            "Dia de Todos os Santos", (f"{year}-11-01" for year in range(1976, 2050))
+        )
+
+    def test_christmas_day(self):
+        self.assertHolidayName("Dia do Natal", (f"{year}-12-25" for year in range(1976, 2050)))
+
+    def test_holy_thursday(self):
+        name = "Quinta-Feira Santa"
+        dt = (
+            "2021-04-01",
+            "2022-04-14",
+            "2023-04-06",
+            "2024-03-28",
+            "2025-04-17",
+        )
+        self.assertHolidayName(name, self.opt_holidays, dt)
+        self.assertHolidayName(name, self.opt_holidays, range(1976, 2050))
+        self.assertNoHolidayName(name)
+        self.assertNoHoliday(dt)
```

Comment:
```suggestion
        self.assertNoHolidayName(name)
        self.assertHolidayName(
            name,
            self.opt_holidays,
            "2021-04-01",
            "2022-04-14",
            "2023-04-06",
            "2024-03-28",
            "2025-04-17",
        )
        self.assertHolidayName(name, self.opt_holidays, range(1976, 2050))
```
Let's use more standard sequence. And about dates range. For the holidays above it was 2019-2025, here it is 2021-2025. I suggest to use the standard 2020-2025 everywhere.